src/Sylius/Bundle/CoreBundle/spec/EventListener/OrderPromotionListenerSpec.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
```

Comment:
You'll have to add the standard Sylius file header to any files you add for the Sylius project